gcp-cloud-run/50-cloud-run.tf

```diff
@@ -1,122 +1,121 @@
-
 ################################
 ### Cloud Run (Vault Server) ###
 ################################
 
 locals {
   pvault_region = coalesce(var.pvault_region, var.default_region)
-
-  env_keys = tolist(keys(var.pvault_env_vars))
+  # Override certain variables
+  pvault_env = merge(
+    var.pvault_env_vars,
+    {
+      "PVAULT_LOG_CUSTOMER_IDENTIFIER" = var.pvault_log_customer_identifier,
+      "PVAULT_LOG_CUSTOMER_ENV"        = var.pvault_log_customer_env,
+      "PVAULT_LOG_CUSTOMER_REGION"     = var.pvault_region,
+      "PVAULT_TLS_ENABLE"              = "false",
+      "PVAULT_DB_REQUIRE_TLS"          = "false",
+      "PVAULT_DB_HOSTNAME"             = module.postgresql-db.private_ip_address,
+      "PVAULT_DB_NAME"                 = var.cloudsql_name,
+      "PVAULT_DB_USER"                 = var.cloudsql_username,
+      "PVAULT_SERVICE_LICENSE"         = var.pvault_service_license,
+      "PVAULT_KMS_URI"                 = "gcp-kms://${google_kms_crypto_key.vault-encryption-key.key_ring}/cryptoKeys/${google_kms_crypto_key.vault-encryption-key.name}",
+      "PVAULT_DEVMODE"                 = tostring(var.pvault_devmode),
+    }
+  )
 }
 
-resource "google_cloud_run_service" "pvault-server" {
+resource "google_cloud_run_v2_service" "pvault-server" {
   name     = "${var.deployment_id}-pvault-server"
   location = local.pvault_region
-  provider = google-beta
+  client   = "terraform"
```

Comment:
Is this a common value for this input?

Reply:
Was added before as an annotation: `run.googleapis.com/client-name`
It helps to confirm if any operation was done through another method (console, api, etc) by triggering a diff on the field.